octodns_digitalocean/__init__.py

```diff
@@ -10,7 +10,7 @@
 from octodns.provider import ProviderException
 from octodns.provider.base import BaseProvider
 
-__VERSION__ = '0.0.1'
+__VERSION__ = '0.0.2'
```

Comment:
You want to try doing a release once this is merged since the version is bumped and changelog updated?

Reply:
Sure, would love to